smp/os_management.py

```diff
@@ -73,7 +73,7 @@ class TaskStatisticsReadRequest(message.ReadRequest):
 class TaskStatistics(BaseModel):
     """Task statistics."""
 
-    model_config = ConfigDict(extra="forbid", frozen=True)
+    model_config = ConfigDict(extra="allow", frozen=True)
```

Comment:
I'm hesitant to loosen the validation and type safety here. The specification lists all fields, so it seems that it should be an implementation error if the SMP server sends unknown fields.

A compromise would be for the tasks field to be a type union of TaskStatistics and "Unknown", so that users can get type safety for those SMP servers that do conform to the spec, and otherwise provide their own deserialization of the Unknown mapping.